README.md

```diff
@@ -1 +1,189 @@
-# -portfolio-conference-page-v2
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+
+  
+  <br/>
+
+  <h3><b>HTML/CSS & JavaScript capstone project - Conference page</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Capstone 1: Conference page] <a name="Capstone 1: Conference page"></a>
+
+**[Capstone]** This is my capstone of module 1.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- HTML, CSS, GIT, Javascript, Github
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Index Page]**
+- **[About Page]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Deploy Page Link](https://alex1779.github.io/-portfolio-conference-page-v2/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] As said in the requirement, please add the recorded video in the readme file explaining some of your key features and proud sections of the project